docs: Add CLI docs for 0.16.1 and 0.17.0 (#1300)
keptn/keptn.github.io: content/docs/0.17.x/reference/cli/commands/keptn_trigger_delivery.md

### Examples

```
keptn trigger delivery --project=<project> --service=<service> --image=<image[:tag]> [--sequence=<sequence>]
```

### Options

```
      --image string            The image name, e.g.
                                "docker.io/<YOUR_ORG>/<YOUR_IMAGE> or quay.io/<YOUR_ORG>/<YOUR_IMAGE>

>                                 "Optionally, you can append a tag using ":<YOUR_TAG>. If no tag is provided, "latest" will be used per default

Reviewer: ```suggestion
                                "Optionally, you can append a tag using ":<YOUR_TAG>. If no tag is provided, the default value of "latest" is used
```
